src/metabase/sync/interface.clj

```diff
@@ -12,7 +12,10 @@
    [:schema         [:maybe ::lib.schema.common/non-blank-string]]
    [:require-filter {:optional true} :boolean]
    ;; `:description` in this case should be a column/remark on the Table, if there is one.
-   [:description    {:optional true} [:maybe :string]]])
+   [:description    {:optional true} [:maybe :string]]
+   [:properties     {:optional true} [:maybe
+                                      [:map {:closed true}
```

Comment:
I think it's a bit of a taste, IMO sync interface should all be closed as it forces us to keep the schema up-to-date as it's easy to forget about it :)

Reply:
In this case it might be annoying if we try to work on an old branch with a new database. I don't think that trumps your reasoning, I was just curious, and in any case also a conservative clam when it comes to typing sensibilities 😄

Reply:
I prefer leaving maps open, that way drivers can be backwards compatible with older versions if we introduce a new key somewhere.